Source/MonoGame.Extended/FramesPerSecondCounter.cs

```diff
@@ -4,9 +4,10 @@
 
 namespace MonoGame.Extended
 {
-    public class FramesPerSecondCounter : IUpdate
+    public class FramesPerSecondCounter : DrawableGameComponent
     {
-        public FramesPerSecondCounter(int maximumSamples = 100)
+        public FramesPerSecondCounter(Game game, int maximumSamples = 100)
+            :base(game)
```

Comment:
One thing that's always bothered me about game components is the way they take a dependency on the `Game` class for no good reason.